thentos-core/src/Thentos/Types.hs

```diff
@@ -507,6 +509,22 @@ instance ToJSON ByUserOrServiceId where
     toJSON (ByService i k) = Aeson.object [ "service" .= Aeson.toJSON (i, k)]
 
 
+-- * send email
+
+-- TODO: Service ID? Pseudonyms? From address?
```

Comment:
Ah, you were asking if we want the sender to be able to change that: no, default sender from the config is fine.